content/en/security/application_security/enabling/compatibility/nodejs.md

```diff
@@ -6,46 +6,153 @@ type: multi-code-lang
-| Vulnerability Management <br/> --> Open source vulnerability detection <br/> | <br/> 2.23.0 for NodeJS 12+, or 3.10.0 for NodeJS 14+ |
-| Vulnerability Management <br/> --> Custom code vulnerability detection <br/> | <br/> 2.32.0 for NodeJS 12+, or 3.19.1 for NodeJS 14+ |
 
-The minimum tracer version to get all supported ASM capabilities for Node.js is 3.19.1.
 
-**Note**: Threat Protection requires enabling [Remote Configuration][2], which is included in the listed minimum tracer version.  
 
 <div class="alert alert-info">If you would like to see support added for any of the unsupported capabilities or for your Node.js framework, let us know! Fill out <a href="https://forms.gle/gHrxGQMEnAobukfn7">this short form to send details</a>.</div>
 
 
+### Networking framework compatibility
+
+
+**Networking tracing provides:**
+
+- Distributed tracing through your applications
+- Request-based blocking
+
+##### ASM Capability Notes
+- Open Source Vulnerability Detection is supported on all frameworks
+
+
+
+| Framework                | Threat Detection supported? | Threat Protection supported? | Custom Code Vulnerability Detection supported? |
+| ------------------------ | ----------- | --------------- | ---------------------------------------------- | ---------------------------------------------- |
+| dns     | {{< X >}} |  |  |
+| fs      |  {{< X >}} |  |  {{< X >}}  |
+| http    |  {{< X >}} |  {{< X >}}  |  {{< X >}}  |
+| https   |  {{< X >}} |  {{< X >}} |  |
+| http2   |  {{< X >}} |  {{< X >}} |  |
+
+
+<div class="alert alert-info">If you don't see your framework of choice listed, let us know! Fill out <a href="https://forms.gle/gHrxGQMEnAobukfn7">this short form to send details</a>.</div>
+
+### Data store compatibility
+
+
+**Datastore tracing provides:**
+
+- Timing request to response
+- Query info (for example, a sanitized query string)
+- Error and stacktrace capturing
+
+##### ASM Capability Notes
+- Open Source Vulnerability Detection is supported on all frameworks
+- Threat Protection works at the HTTP request (input) layer, and so works for all databases by default
+
+### Data store compatibility
+
+| Framework                 | Versions | Threat Detection supported? | Threat Protection supported? | Custom Code Vulnerability Detection supported? |
+| ----------------------- | ---------- | --------------- | ---------------------------------------------- | ---------------------------------------------- |
+| [cassandra-driver][28] | `>=3`    |{{< X >}}|{{< X >}} |          |
+| [couchbase][29]        | `^2.4.2` |{{< X >}}|          |          |
+| [elasticsearch][30]    | `>=10`   |{{< X >}}|          |          |
+| [ioredis][31]          | `>=2`    |{{< X >}}|{{< X >}} |          |
+| [knex][32]             | `>=0.8`  |{{< X >}}|{{< X >}} |          |
+| [mariadb][63]          | `>=3`    |{{< X >}}|{{< X >}} |          |
```

Comment:
Hi @sourabhkatti the link here (63) doesn't exist so this needs updated with the correct link.
I fixed a few other typos and broken links but wasn't sure what the correct on is here.
Once this is fixed, it should be good to go!